src/util/otpReminder.tsx

```diff
@@ -13,72 +13,71 @@ const OTP_REMINDER_KEY_NAME_LAST_OTP_CHECKED = 'lastOtpCheck'
 const OTP_REMINDER_KEY_NAME_DONT_ASK = 'OtpDontAsk'
 
 /**
- * Check and show the 2fa reminder on login.
+ * Check and return a potential 2fa reminder modal if needed
  */
-export async function showOtpReminder(account: EdgeAccount): Promise<void> {
+export async function getOtpReminderModal(account: EdgeAccount): Promise<(() => Promise<void>) | undefined> {
   const { otpKey, dataStore, username, created } = account
-  if (username == null) return
-  if (otpKey != null) return
-
+  if (username == null || otpKey != null) return
   const [dontAsk, lastOtpCheckString]: [string | null, string | null] = (await Promise.all([
     dataStore.getItem(OTP_REMINDER_STORE_NAME, OTP_REMINDER_KEY_NAME_DONT_ASK).catch(() => null),
     dataStore.getItem(OTP_REMINDER_STORE_NAME, OTP_REMINDER_KEY_NAME_LAST_OTP_CHECKED).catch(() => null)
   ])) as any
-
   if (dontAsk) return
   const lastOtpCheck = lastOtpCheckString != null ? parseInt(lastOtpCheckString) : null
-
-  // Show the modal if we have never shown it before,
```

Comment:
These comments are still helpful, since the `if` statements are pretty gnarly. Maybe replace the first word "Show" with "Return", but otherwise I think they should stay.

Reply:
Added em back slightly reworded